test/data/data_test.go

```diff
@@ -0,0 +1,40 @@
+// Protocol Buffers for Go with Gadgets
+//
+// Copyright (c) 2013, The GoGo Authors. All rights reserved.
```

Comment:
Not really, I just looked at the other files and they all seemed to have one of these tests, I'll remove it.